repos/system_upgrade/el7toel8/files/rhel_upgrade.py

```diff
@@ -49,9 +49,9 @@ def configure(self):
         self.base.repos.all().disable()
         for repo in self.base.repos.all():
             if repo.id in enabled_repos:
+                repo.skip_if_unavailable = False
```

Comment:
In case some of the repositories are unavailable and are skipped, probably the transaction calculation will fail. In current case I would even expect that all repositories with be either available or not. Shouldn't we warn the user if any repository is not available?

Reply:
I agree. Definitely there should be some message about what is the problem. But this is still better then original state and we do not have time now for something better. So I am going to create issue and merge this.I tested that just now and it works as expected.

Reply:
@artmello, at least with this change, dnf will end execution of the upgrade plugin right away with a message that the required repo(s) is not available.